src/main/java/com/cinema/mapper/GenericMapper.java

```diff
@@ -0,0 +1,9 @@
+package com.cinema.mapper;
+
+public interface GenericMapper<T, K, V> {
```

Comment:
To my opinion this interface and its children are redundant, classes would be enough

Reply:
I agree with @Rommelua GenericMapper interface is redundant
we can use only classes